File: python/vedant_bhatnagar/DevsnestProblems_dailyUpdated/twoSumSorted.py
def solve(n, arr, target):
    # CODE HERE
    low = 0
    high = n-1
    sum = 0
    while(low<high):
        sum = 0
        sum = arr[low] + arr[high]
        if target > sum:
            low += 1
        elif target < sum:
            high -= 1
        else:
            return [low+1, high+1] 

File: java/ankit_kanyal/devnest_problems/phase_1/TwoSumSorted.java
// link- https://devsnest.in/algo-challenges/two-sum-ii

static int[] solve(int n, int[] arr, int target){
//CODE HERE 
int start=0,end=n-1;
while(start<end){
    int sum=arr[start]+arr[end];
    
    if(sum==target) return new int[]{start+1,end+1};
    else if(sum<target){
        start++;
    }
    else end--;
}
return new int[]{-1,-1};
}